src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -52,7 +52,10 @@ def reset(self):
         Delete the content of the inventory table.
         The next call to `snapshot` will re-populate the table.
         """
-        self._exec(f"DELETE FROM {self.full_name}")
+        try:
+            self._exec(f"DELETE FROM {self.full_name}")
```

Comment:
Looks out of scope?

Reply:
this is needed - if the table does not exist, then reset() would crash